### PR TITLE
VRay DR fixes

### PR DESCRIPTION
**Pull request by smith1511**
Updates to pre-render script creation.
Fixes for VRay DR (create spawner config and run as admin)

### CODE REVIEW

File: ncj/3dsmax/vray-dr/job.template.json [hunk 1]
```diff
@@ -20,13 +20,12 @@
         },
         "maxVersion": {
             "type": "string",
-            "defaultValue": "2019",
+            "defaultValue": "2018",
```
Comment (matthew-rollitt) [inline ncj/3dsmax/vray-dr/job.template.json:23]: Why did you change default down to 2018, we made the choice to default over to the newest version.

Comment (matthew-rollitt) [reply ncj/3dsmax/vray-dr/job.template.json:23]: Based on the "allowedValues" below I take it this doesn't work on 2019

File: ncj/3dsmax/vray-dr/pool.template.json [hunk 2]
```diff
@@ -25,42 +25,25 @@
             "metadata": {
                 "description": "The size of the virtual machines that run the application"
             },
-            "defaultValue": "Standard_D2_v2",
+            "defaultValue": "Standard_F8s_v2",
             "allowedValues": [
```
Comment (matthew-rollitt) [inline ncj/3dsmax/vray-dr/pool.template.json:29]: why is the default F8s_V2?

**Review by matthew-rollitt — COMMENTED** [review]
Just curious about 2019

File: ncj/3dsmax/vray-dr/job.template.json (hunk 1 above)
Comment (smith1511) [reply ncj/3dsmax/vray-dr/job.template.json:23]: This is just for DR - 2019 isn't supported because it doesn't have Backburner installed.

File: ncj/3dsmax/vray-dr/pool.template.json (hunk 2 above)
Comment (smith1511) [reply ncj/3dsmax/vray-dr/pool.template.json:29]: It's probably better to default to a VM that has better capabilities as the D2s aren't really usable...
